atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/ThreadLocalWrappingTransactionManager.java

```diff
@@ -1,41 +0,0 @@
-/**
- * Copyright 2015 Palantir Technologies
- *
- * Licensed under the BSD-3 License (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://opensource.org/licenses/BSD-3-Clause
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.palantir.atlasdb.transaction.impl;
-
-import com.palantir.atlasdb.transaction.api.LockAwareTransactionManager;
-import com.palantir.atlasdb.transaction.api.Transaction;
-import com.palantir.atlasdb.transaction.api.TransactionWrapper;
-import com.palantir.common.concurrent.ExecutorInheritableThreadLocal;
-
-public class ThreadLocalWrappingTransactionManager extends WrappingTransactionManager {
```

Comment:
I agree, these should be made consistent and will help move use towards the work of all internal products using the same code paths.